benchmark.rb

```diff
@@ -0,0 +1,26 @@
+require 'tty'
+require 'json'
+require_relative 'lib/lens/gzip_util'
+require_relative 'lib/lens/compression'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.